bucket/lazarus.json

```
@@ -0,0 +1,50 @@
+{
+    "homepage": "https://www.lazarus-ide.org",
+    "version": "2.0.0",
+    "description": "A Delphi compatible cross-platform IDE for Rapid Application Development.",
+    "license": {
+        "identifier": "GPL-2.0-or-later | FPC-modified-LGPL-2.0-or-later",
+        "url": "https://spdx.org/licenses/GPL-2.0-or-later.html; http://wiki.freepascal.org/FPC_modified_LGPL"
+    },
+    "architecture": {
+        "64bit": {
+            "url": "https://downloads.sourceforge.net/project/lazarus/Lazarus%20Windows%2064%20bits/Lazarus%202.0.0/lazarus-2.0.0-fpc-3.0.4-win64.exe",
+            "hash": "e53f18e15ff94998cda968325214d11326f20f148b47f349230151dbfc11ca70"
+        },
+        "32bit": {
+            "url": "https://downloadssourceforge.net/project/lazarus/Lazarus%20Windows%2032%20bits/Lazarus%202.0.0/lazarus-2.0.0-fpc-3.0.4-win32.exe",
+            "hash": "fb64be2210c3583f3a184a4a3acc2342624582f7d946ab06d4faf64c81e0326d"
+        }
+    },
+    "innosetup": true,
+    "post_install": [
+        "$fpcdir = \"$dir\\fpc\\\" + ([Regex]'fpc-([\\d.]+)').Matches($fname).Groups[1].Value",
```

Is it needed to call regex library? Something like `-replace '..', '$1' should work too.
```suggestion
        "$fpcdir = \"$dir\\fpc\\\" + ("$fname" -replace 'fpc-([\\d\\.]+)', '$1')
```

---

Thanks, `-replace` does works.